### ISSUE TITLE
Checker - Keyboard focus on the scan drop down button is not clear.

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#1107]
- [X] `a11y Checker`
- IBM Accessibility checker Version 3.1.37
- Keyboard FF

Page element: Scan dropdown.
Keyboard focus indicator on the element should be darker.

Triage: Dev will investigate if we're using the latest carbon component correctly.  

Ind Bug - moved from internal https://github.ibm.com/IBMa/E2E/issues/4996

**Comment by drjoho** [comment]
Will discuss with design to get a better color and separation for the "scan drop down button"

**Comment by marcjohlic** [comment]
Josiah will be tackling this as a part of #1106  -  closing
